test/infrastructure/docker/api/v1alpha3/dockermachine_types.go

```diff
@@ -79,6 +80,14 @@ type DockerMachineStatus struct {
 	// added to the load balancer
 	// +optional
 	LoadBalancerConfigured bool `json:"loadBalancerConfigured,omitempty"`
+
+	// ImagePreLoaded denotes that images where pre-loaded into the machine
+	// +optional
+	ImagePreLoaded bool `json:"imagePreLoaded,omitempty"`
```

Comment:
This change is not strictly related to this PR, but it helps to get the code re-entrant in case of problems during container provisioning